From 38d1d22b1d4cf39d865045b5368d6c6b896f9ba1 Mon Sep 17 00:00:00 2001
From: Nickey Yang <nickey.yang@rock-chips.com>
Date: Sun, 5 Apr 2020 14:51:49 +0800
Subject: [PATCH] input: touchscreen: gt1x: update config for support GT917S

Old panel combination: st7703 + gt5688 use GTP_CFG_GROUP1
New panel combination: ili9881d + gt917s use GTP_CFG_GROUP0
Tested on rk3326 rv1108 platform.

Signed-off-by: Nickey Yang <nickey.yang@rock-chips.com>
Change-Id: I6af571f5625316f3418cb51673fc3e43317dff74
---
 drivers/input/touchscreen/gt1x/gt1x_generic.h | 44 +++++++++----------
 1 file changed, 20 insertions(+), 24 deletions(-)

diff --git a/drivers/input/touchscreen/gt1x/gt1x_generic.h b/drivers/input/touchscreen/gt1x/gt1x_generic.h
index 09721a62dbeb..3f53bfcb35d0 100644
--- a/drivers/input/touchscreen/gt1x/gt1x_generic.h
+++ b/drivers/input/touchscreen/gt1x/gt1x_generic.h
@@ -105,30 +105,26 @@
 #define CFG_GROUP_LEN(p_cfg_grp)  (sizeof(p_cfg_grp) / sizeof(p_cfg_grp[0]))
 /* TODO define your config for Sensor_ID == 0 here, if needed */
 #define GTP_CFG_GROUP0 {\
-    0x44, 0xD0, 0x02, 0x00, 0x05, 0x05, 0x3D, 0x10, 0x00, 0x08, \
-    0x00, 0x05, 0x3C, 0x28, 0x5E, 0x00, 0x11, 0x00, 0x00, 0x00, \
-    0x28, 0x82, 0x96, 0xFC, 0xC8, 0x00, 0x00, 0x00, 0x00, 0x00, \
-    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x10, 0x00, 0x00, \
-    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x87, 0x26, 0x17, 0x64, \
-    0x66, 0xDF, 0x07, 0x91, 0x31, 0x18, 0x0C, 0x43, 0x24, 0x00, \
-    0x06, 0x28, 0x6E, 0x80, 0x94, 0x02, 0x05, 0x08, 0x04, 0xDA, \
-    0x33, 0xAF, 0x3F, 0x92, 0x4A, 0x7F, 0x56, 0x71, 0x62, 0x66, \
-    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, \
-    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x03, 0x0F, 0x19, 0x04, \
-    0x0F, 0x10, 0x42, 0xD8, 0x0F, 0x00, 0x00, 0x00, 0x00, 0x00, \
-    0x00, 0x00, 0xFF, 0xFF, 0x04, 0x00, 0x00, 0x00, 0x00, 0x00, \
-    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, \
-    0x00, 0x00, 0x00, 0x05, 0x1E, 0x00, 0x70, 0x17, 0x50, 0x1E, \
-    0x08, 0x09, 0x0A, 0x0B, 0x0C, 0x0F, 0x0E, 0x10, 0x0D, 0x12, \
-    0x13, 0x1F, 0x1E, 0x1D, 0x1C, 0x1B, 0x1A, 0x19, 0x18, 0x17, \
-    0x16, 0x15, 0x14, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, \
-    0xFF, 0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08, \
-    0x09, 0x0A, 0x0B, 0x0C, 0x0D, 0xFF, 0xFF, 0xFF, 0xFF, 0x00, \
-    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, \
-    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, \
-    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xC8, 0x00, 0x00, \
-    0x00, 0x00, 0x24, 0x1E, 0x6D, 0x00, 0x14, 0x28, 0x00, 0x00, \
-    0x10, 0x00, 0x00, 0x00, 0x00, 0x00, 0x23, 0x99, 0x01\
+	0x45,0xD0,0x02,0x00,0x05,0x05,0x3D,0x04,0x08,0xC1,0x0A,0x0F,\
+	0x5F,0x3C,0x53,0x01,0x00,0x32,0x00,0x00,0x1E,0x8C,0xA0,0xC8,\
+	0x18,0x04,0x00,0x00,0x00,0x00,0x3C,0x00,0x00,0x22,0x00,0x00,\
+	0x00,0x00,0x00,0x40,0x00,0x00,0x00,0x00,0x38,0x28,0x86,0x26,\
+	0x16,0x53,0x55,0xC5,0x04,0x28,0x3C,0x28,0x3D,0x42,0x24,0x00,\
+	0x03,0x32,0x78,0xC0,0x94,0xF2,0x00,0x00,0x53,0xCC,0x3D,0xA9,\
+	0x49,0x91,0x54,0x80,0x60,0x71,0x6C,0x66,0x00,0x00,0x00,0x00,\
+	0x00,0x00,0x00,0xF0,0x50,0x3C,0xAA,0xAA,0x27,0x3C,0x32,0x00,\
+	0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x20,0x00,\
+	0x02,0x78,0x0A,0x50,0xFF,0xE4,0x04,0x00,0x00,0x00,0x00,0x00,\
+	0x00,0x00,0x00,0x3D,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,\
+	0x00,0x00,0x00,0x00,0x1D,0x0D,0x50,0x1E,0x13,0x14,0x15,0x16,\
+	0x17,0x18,0x19,0x1A,0x1B,0x1D,0x1F,0x12,0x10,0x0F,0x0E,0x0D,\
+	0x0C,0x0B,0x0A,0x09,0x08,0x07,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,\
+	0xFF,0xFF,0xFF,0xFF,0x06,0x08,0x0C,0x12,0x13,0x14,0x15,0x17,\
+	0x18,0x19,0x1A,0x1B,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,\
+	0xFF,0xFF,0x00,0x30,0x00,0x00,0x78,0x0F,0x1E,0x64,0x00,0x00,\
+	0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,\
+	0x00,0x11,0x00,0x00,0x32,0x03,0x14,0x50,0x0C,0x02,0x14,0x50,\
+	0x00,0x55,0x10,0x00,0x3C,0xAD,0x06,0x3C,0x45,0x07,0x01 \
 }
 /* TODO define your config for Sensor_ID == 1 here,  if needed */
 #define GTP_CFG_GROUP1 {\
-- 
2.35.3

